File: cekhoki.cpp
#include <iostream>

void cekHoki(int n) {
    if (n % 2 == 0 && n % 7 == 0) {
        std::cout << "HOKI" << std::endl;
    } else {
        std::cout << "BIASA" << std::endl;
    }
}

int main() {
    cekHoki(14); 
    cekHoki(10); 
    return 0;
}

File: cekhoki.py
def cekHoki(n):
    if n % 2 == 0 and n % 7 == 0:
        print("HOKI")
    else:
        print("BIASA")

cekHoki(14)  # Output: HOKI
cekHoki(10)  # Output: BIASA
